src/Node.js

```diff
@@ -0,0 +1,48 @@
+  // ### DOM Node Shim
+  //
+  // This represents a superficial form of a DOM node which contains most of
+  // the properties that a regular DOM node would contain, but only the ones
+  // needed by slowparse.
+module.exports = (function(){
+  "use strict";
+
+  function Node(nodeType, nodeName, nodeValue, namespaceURI) {
+    this.nodeType = nodeType || null;
```

Comment:
the following three bits of information are always necessary. I'd say don't use fallbacks for them. Let the code crash if they're missing.